packages/venia-concept/src/index.js

```diff
@@ -20,6 +21,21 @@ import { registerSW } from './registerSW';
 const { BrowserPersistence } = Util;
 const apiBase = new URL('/graphql', location.origin).toString();
 
+const language = STORE_VIEW_LOCALE;
+let messages;
+
+try {
+    messages = require(`@magento/venia-ui/lib/i18n/${language.replace(
+        '-',
+        '_'
+    )}.json`);
```

Comment:
What do you think about making 2 utility functions? one to covert from Magento to `react-intl` and one to convert it back?